### PR TITLE
feat: Selectable files from the media library

### PR DESCRIPTION
**Pull request by DavidKelemen**

### CODE REVIEW

**Review by SoetZombie — APPROVED** [review]
LGTM, couldn't test in SC enviroment byt code changes looks fine 👌👌

File: src/Ucommerce.Sitecore/Content/SitecoreImageService.cs [hunk 1]
```diff
@@ -42,11 +43,39 @@ public virtual Ucommerce.Content.Content GetImage(string contentId)
...
+
+		protected virtual Ucommerce.Content.Content MapItemToContent(Item item)
+		{
+			return new Ucommerce.Content.Content()
+			{
+				Name = item.Name,
+				Icon = item.Appearance.Icon,
+				Url = GetMediaUrl(item),
+				Id = item.ID.ToString(),
+				// NodeType = item.Template.Key == "media folder" ? "Folder" : "Image"
```
Comment (ilny) [inline src/Ucommerce.Sitecore/Content/SitecoreImageService.cs:57]: Has this been left in place intentionally as a reminder for some sort of next step of the fix/refactoring?
If not, please remove it.

Comment (SoetZombie) [reply src/Ucommerce.Sitecore/Content/SitecoreImageService.cs:57]: Was about to write the same comment 😂

File: src/Ucommerce.Sitecore/Content/SitecoreImageService.cs [hunk 2]
```diff
@@ -42,11 +43,39 @@ public virtual Ucommerce.Content.Content GetImage(string contentId)
...
+		}
+
+		protected virtual string GetNodeType(Item item)
+		{
+			var folderTemplateKeys = new[] {"media folder", "node"};
+			var imageTemplateKey = "image";
+
+			if (folderTemplateKeys.Contains(item.Template.Key))
+			{
+				return "Folder";
```
Comment (SoetZombie) [inline src/Ucommerce.Sitecore/Content/SitecoreImageService.cs:69]: Can we use constants for these?

File: src/Ucommerce.Sitecore/Content/SitecoreImageService.cs (hunk 1 above)
Comment (DavidKelemen) [reply src/Ucommerce.Sitecore/Content/SitecoreImageService.cs]: Yeah, I thought I made this a draft (no matching story in the review column on CH at the time.)

**Pull request merged**